Assignment5/Assignment5_QNo1/module_sub.rb

```diff
@@ -0,0 +1,10 @@
+# => Write a program to define two modules and both having a method
+# => with same name but with different functionality and then define
+# => a method in a class and use those module methods inside that.
+module ModuleSub
+  # => define common method
+  def self.common_method
+    puts "The subtraction of two numbers 9 - 5 = 4"
+  end
```

Comment:
@HrisikeshSinghaChowdhury Instead of using self here how can you use this method as a class method inside the class. There is an option to extend the module inside a class , Can you use that to achieve this?